cluster/src/main/java/com/hartwig/pipeline/metadata/ApiFileOperation.java

```diff
@@ -0,0 +1,8 @@
+package com.hartwig.pipeline.metadata;
+
+import com.hartwig.pipeline.sbpapi.FileResponse;
+import com.hartwig.pipeline.sbpapi.SbpRestApi;
+
+public interface ApiFileOperation {
```

Comment:
I like the thought behind this but feel like it introduces inconsistency to only have ApiFile operation and no similar abstractions for other operations.